### ISSUE TITLE
tooltips accessibility

### ISSUE THREAD

**Issue by NeilWix** [Fyrd/caniuse#6430]
The site tooltips aren't WCAG compliant https://www.w3.org/TR/WCAG22/#content-on-hover-or-focus
Users that use a magnifier need to move the cursor over the tooltip body in order to read the content. Currently doing this collapses the tooltip

**Comment by Fyrd** [comment]
Assuming you're referring to the version support cell tooltips, you can click on a cell (or tab to it) and the tooltip should open without automatically collapsing. Is this sufficient or is there another change you'd recommend?

**Comment by NeilWix** [comment]
Yes, I am referring to the version support cell tooltip. I guess this kind of suffice, though I must say, it's somewhat unconventional. The primary audience for this WCAG guideline are screen magnifier users (see screen shot that illustrates the need to move the mouse over the tooltip). I myself use the site for years, get annoyed by not being able to move the pointer over the tooltip - not knowing there is an option to click the cell...
So, now looking a little bit more into it, I also notice the following:
**Sighted keyboard users** can not tab into the tooltip [interactive] content. For this, the tooltip should be next in DOM, or alternatively you need to preventDefault and override the default tab key target (a bit messy).
Screen reader users are not exposed at all to the tooltip content
![image](https://user-images.githubusercontent.com/50541527/186184218-1da1d2ce-2244-4a98-a667-1861228f963e.png)

**Comment by Fyrd** [comment]
Thanks for sharing! I definitely see the issue now and have pushed a fix. Still seeing some issues in Firefox but hopefully the new tab behavior is on the right track.

**Comment by Fyrd** [comment]
Turns out the Firefox issue I noticed was due [this MacOS quirk](https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox) so no further work needed there. 

@NeilWix were there any other related issues or can this ticket be closed?

**Comment by NeilWix** [comment]
No, thanks for the update
